ZConfig/components/logger/loghandler.py

```diff
@@ -25,6 +25,13 @@
 from logging.handlers import HTTPHandler, SMTPHandler
 from logging.handlers import NTEventLogHandler as Win32EventLogHandler
 
+# Export these, they're used in handlers.py
+SysLogHandler = SysLogHandler
+HTTPHandler = HTTPHandler
+SMTPHandler = SMTPHandler
+Win32EventLogHandler = Win32EventLogHandler
```

Comment:
This applies to this comment and two of the previous ones:

linters like flake8 and pylint (I run/ran both during this process) complain (rightfully) about unused imports. The way to fix that without adding `#noqa` or what have you is to *use* the import, and the simplest way to do that is to rebind it to the same name. This declarative statement seems reasonably clear to me that this name is a public part of the module definition, as opposed to a side-effect of the import.

In this particular case, it seemed important to leave the imports as they are for backwards compatibility.